external.bzl

```diff
@@ -1116,12 +1116,11 @@ def _extractor_image_dependencies():
     """Defines external repositories necessary for extractor images."""
     maybe(
         http_archive,
-        name = "com_github_philwo_bazelisk",
-        sha256 = "cb6a208f559fd08d205527b69d597ef36f7e1a922fe1df64081e52dd544f7666",
-        strip_prefix = "bazelisk-0.0.2",
+        name = "com_github_bazelbuild_bazelisk",
+        sha256 = "a38d364a193cedd0b4623aa61463e118c018013f0787e368cdf52139ce21b4ed",
+        strip_prefix = "bazelisk-v1.3.0",
```

Comment:
This prefix is incorrect, it should be `bazelisk-1.3.0` (without the `v`). This is causing my `git push` to fail:
INFO: Repository 'remote_java_tools_linux' used the following cache hits instead of downloading the corresponding file.
 * Hash '3ff465e82954a70f49982610dd63f6f651beaa83c707dd637870b0e41cdcd2f0' for https://mirror.bazel.build/bazel_java_tools/releases/javac11/v7.0/java_tools_javac11_linux-v7.0.zip
If the definition of 'remote_java_tools_linux' was updated, verify that the hashes were also updated.
ERROR: /usr/local/google/home/shahms/src/kythe/kythe/extractors/bazel/BUILD:3:1: no such package '@com_github_bazelbuild_bazelisk//': java.io.IOException: Error extracting /usr/local/google/home/shahms/.cache/bazel/_bazel_shahms/f0c8a49cbf1bfaa82de86be744427f4b/external/com_github_bazelbuild_bazelisk/v1.3.0.zip to /usr/local/google/home/shahms/.cache/bazel/_bazel_shahms/f0c8a49cbf1bfaa82de86be744427f4b/external/com_github_bazelbuild_bazelisk: Prefix "bazelisk-v1.3.0" was given, but not found in the archive. Here are possible prefixes for this archive: "bazelisk-1.3.0". and referenced by '//kythe/extractors/bazel:docker'
ERROR: Evaluation of subquery "rdeps(//..., set(//kythe/extractors/bazel:Dockerfile //kythe/extractors/bazel:BUILD //kythe/cxx/indexer/cxx/testdata:basic/vname_corpus.json //kythe/cxx/indexer/cxx/testdata:basic/vname_corpus.cc //kythe/cxx/indexer/cxx/testdata:basic/namespace_vnames.cc //kythe/cxx/indexer/cxx/testdata:BUILD //kythe/cxx/indexer/cxx:KytheIndexerMain.cc //kythe/cxx/indexer/cxx:KytheGraphObserver.h //kythe/cxx/indexer/cxx:KytheGraphObserver.cc //kythe/cxx/indexer/cxx:IndexerFrontendAction.h //kythe/cxx/indexer/cxx:IndexerFrontendAction.cc //kythe/cxx/indexer/cxx:IndexerASTHooks.cc //kythe/cxx/extractor:cxx_extractor.cc //:external.bzl))" failed (did you want to use --keep_going?): errors were encountered while computing transitive closure
Loading: 894 packages loaded
Loading: 894 packages loaded

Reply:
Fix in https://github.com/kythe/kythe/pull/4420.

Reply:
Followed by ` no such package '@com_github_mitchellh_go_homedir//': The repository '@com_github_mitchellh_go_homedir' ` after fixing that.

Reply:
Thanks, #4420 should fix that issue too.